keyboards/sawnsprojects/eclipse/eclipse60/info.json

```diff
@@ -0,0 +1,117 @@
+{
+    "keyboard_name": "Eclipse60",
+    "manufacturer": "Salane",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x534C",
+        "pid": "0xE160",
+        "device_version": "1.0.0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "rows": ["B1", "B2", "B7", "B3", "D1", "D0", "F6", "F7", "F4", "F5"],
+        "cols": ["C7", "C6", "B6", "B5", "B4", "D7", "D6"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": true,
+        "audio": false,
+        "key_lock": false
+    },
+    "rgblight": {
+        "pin": "B0",
+        "led_count": 18,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
+        "sleep": true,
+        "animations": {
+            "all": true
+        }
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
`LAYOUT_all` is generally used for boards that support multiple layouts. `LAYOUT` should be used here instead. Same goes for `keyboards/sawnsprojects/eclipse/tinyneko`.